base_layer/wallet/src/transaction_service/protocols/transaction_receive_protocol.rs

```diff
@@ -223,12 +223,15 @@ where TBackend: TransactionBackend + Clone + 'static
                         match rx.await {
                             Ok(send_msg_response) => {
                                 if let SendMessageResponse::Queued(send_states) = send_msg_response {
-                                    debug!("Discovery of {} completed for TxID: {}", self.source_pubkey, self.id);
+                                    debug!(
+                                        target: LOG_TARGET,
+                                        "Discovery of {} completed for TxID: {}", self.source_pubkey, self.id
+                                    );
                                     direct_send_result = self.wait_on_dial(send_states).await;
                                 }
                             },
                             Err(e) => {
-                                error!(
+                                warn!(
```

Comment:
Discovery failed: this could (should) generally be because the peer is offline. Since the fallback (store and forward) is in place, I'd say this could be debug level.

Reply:
fair enough